dotCMS/src/test/java/com/liferay/util/ValidatorTest.java

```diff
@@ -0,0 +1,36 @@
+package com.liferay.util;
+
+import com.dotcms.UnitTestBase;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Oscar Arrieta on 6/19/17.
+ */
+public class ValidatorTest extends UnitTestBase {
```

Comment:
Approved but just a comment, the best practice on UnitTest is to try to test just one thing at time, if you do that will be easier to identified the issue or what fails, instead of looking into the method line.